### PR TITLE
json testing. displaying messages

### PR DESCRIPTION
**Pull request by bhargaviks**
I went through week-3's step-3 walkthrough about JSON. I have added a small section in my index.html that fetches and displays the hard-coded messages with the help of Gson. 

to see what I have changed: https://screenshot.googleplex.com/jxMj568ZJSm.png 
Thank you!

### CODE REVIEW

File: portfolio/pom.xml [hunk 1]
```diff
@@ -23,8 +23,16 @@
...
     </dependency>
+
+    <dependency>
+        <groupId>com.google.code.gson</groupId>
+        <artifactId>gson</artifactId>
+        <version>2.8.6</version>
+    </dependency>
   </dependencies>
 
+
```
Comment (isaacwdavis) [inline portfolio/pom.xml:34]: nit: remove extra lines

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -23,11 +25,36 @@
...
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Bhargavi!");   // Added my name instead of  World.
-    response.getWriter().println("I am learning Servlets!");
+    Gson g = new Gson();                                // Imported in pom and above
+    String json = g.toJson(messages);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+
+    /* 
+      THIS BLOCK OF COMMENTS WERE FROM PREVIOUS WALKTHROUGHS.
```
Comment (isaacwdavis) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: Remove or add a TODO

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -54,4 +54,27 @@ function getGreeting() {
...
+  });
+
+}
+
+/** Creates an <li> element containing text. */
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
 }
```
Comment (isaacwdavis) [inline portfolio/src/main/webapp/script.js:80]: nit: Add new line to end of file

**Comment by bhargaviks** [comment]
Resolved the feedback. Thanks! Closing this pull request.

**Pull request merged**